src/System.Transactions/src/System.Transactions.csproj

```diff
@@ -46,18 +46,12 @@
     <Compile Include="System\Transactions\Configuration\ConfigurationStrings.cs" />
     <Compile Include="System\Transactions\Configuration\DefaultSettingsSection.cs" />
     <Compile Include="System\Transactions\Configuration\MachineSettingsSection.cs" />
-    <Compile Include="System\Transactions\Trace\TraceRecord.cs" />
```

Comment:
If we reuse the TraceRecord, we may not be able to use [callermethodname]. Also, the code will have one extra layer of function call and not look like other ETW tracing code. This will increase confusing on other developers.